src/utils/code.ts

```diff
@@ -222,6 +250,8 @@ import { ${iconImports.join(',')} } from "@chakra-ui/icons";`
       : ''
   }
 
+  ${customImports.length ? `${customImports.join(';')}` : ''}
```

Comment:
Just `${customImports.join(';')}` would do i guess

Reply:
okay, I will change that

Reply:
done